tests/PhpSpreadsheetTests/Reader/XlsTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace PhpOffice\PhpSpreadsheetTests\Reader;
+
+use PhpOffice\PhpSpreadsheet\Reader\Xls;
+use PHPUnit\Framework\TestCase;
+
+class XlsTest extends TestCase
+{
+    /**
+     * Test load Xls file without cell reference.
+     */
+    public function testLoadXlsWithoutCellReference()
+    {
+        $filename = './data/Reader/Xls/without_cell_reference.xls';
```

Comment:
We strive to limit the number of binary files in the repository to keep its size down. Can you replace those with by writing them with PhpSpreadsheet and then reloading them instead ? (this is why I suggested functional tests, see `\PhpOffice\PhpSpreadsheetTests\Functional\AbstractFunctional`)